index.js

```diff
@@ -98,10 +98,17 @@
   //  identity :: a -> a
   var identity = function(x) { return x; };
 
-  //  prepend :: a -> Array a -> Array a
-  var prepend = function(x) {
-    return function(xs) {
-      return [x].concat(xs);
+  //  concat_ :: Array a -> Array a -> Array a
+  var concat_ = function(xs) {
+    return function(ys) {
+      return xs.concat(ys);
+    };
+  };
```

Comment:
well, i don't know if it alpha ordering there, if it should ```has``` must be below ```constant``` function. so should i change it too?